app/controllers/decidim/vocdoni/admin/steps_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Vocdoni
+    module Admin
+      # This controller allows to manage the steps of an election.
+      class StepsController < Admin::ApplicationController
+        helper Decidim::ApplicationHelper
+        helper StepsHelper
+        helper_method :elections, :election, :current_step
+
+        def index
+          enforce_permission_to :read, :steps, election: election
+
+          if current_step_form_class
+            @form = form(current_step_form_class).instance(election: election)
+            @form.valid?
+          end
```

Comment:
I see that there's only one form on the `create_election` step, maybe we could simplify a bit? Or we will put more forms on other steps in the future?

Reply:
That's a great question.

I'm planning to be adding a few more Forms and even Commands in the flow. My idea integrating the Vocdoni SDK, as its made in JavaScript is that: 

1. Decidim presents a Form with hidden fields 
2. Vocdoni SDK makes the requests and leave the responses in these fields. It gets sent to the server. 
3. Decidim validates these values with the Form and saves it with a Command. 

You can see an example of this whole flow in #18. Please tell me if you have any better alternative for this 😄 . As we usually not have write endpoints in GraphQL API, I think this would be "cleanish". 

Just thinking out loud, this will be the mechanism for Results for instance.

In the case of the Organization Wallet, as its for the whole organization and not only for a given Election, I actually do it outside Steps as that I didn't wanted to introduce a "step" that only gets called the first time (see #17)

Reply:
Ok, as we are not sure, let's not waste time on this. We'll have plenty of time once we get to the main course dealing with the vocdoni sdk.

Reply:
Ok! Just to propose another alternative that I consider but wasn't that sure that we wanted, is to implement these kind of requests through Node (server side), but that could bring issues in some deployments (for instance, I don't know whats the story with a RoR and Node app in Heroku)